scripts/bundleTs.mjs

```diff
@@ -205,14 +215,17 @@ await Promise.all(
 
     definitionFile = [
       copyrightSnippet,
+      '',
```

Comment:
this newline between the copyright and the contents of the file is needed, otherwise `eslint-plugin-import` bails as it's quite defensive in the case of a comment above it (to avoid breaking `eslint disable` comments).

I also think it reads easier, so win-win 😀